packages/vue/src/components/controlComponents.ts

```diff
@@ -0,0 +1,171 @@
+import type {
+  CheckAuthorizationWithCustomPermissions,
+  HandleOAuthCallbackParams as HandleOAuthCallbackParamsOriginal,
+  OrganizationCustomPermissionKey,
+  OrganizationCustomRoleKey,
+  RedirectOptions,
+} from '@clerk/types';
+import { defineComponent } from 'vue';
+
+import { useAuth } from '../composables/useAuth';
+import { useClerk } from '../composables/useClerk';
+import { useClerkContext } from '../composables/useClerkContext';
+import { useClerkLoaded } from '../utils/useClerkLoaded';
+
+export const SignedIn = defineComponent((_, { slots }) => {
```

Comment:
Would users benefit from us adding JSDoc comments to these exports or wouldn't it be shown in ones IDE?

Reply:
Good question! So the doc shows up in the import statement:

<img width="593" alt="Screenshot 2024-11-07 at 7 51 59 AM" src="https://github.com/user-attachments/assets/46d55e33-24e2-419a-8036-ee3334660df6">

but not when used inside template:

<img width="706" alt="Screenshot 2024-11-07 at 7 52 10 AM" src="https://github.com/user-attachments/assets/029231c2-89ba-4fe5-9d80-b12e72bbe064">

I skipped it because of that but we can definitely add it 🤔

Reply:
Why would it only show up on the import? I guess typically you'd expect this to exist during usage

Reply:
Okay, looks like it is a limitation when using render functions. When doing normal Vue SFC, it works:

<img width="364" alt="Screenshot 2024-11-08 at 7 33 53 AM" src="https://github.com/user-attachments/assets/2c4b165f-b2df-4418-976f-3de71bbe6f34">

<img width="454" alt="Screenshot 2024-11-08 at 7 34 09 AM" src="https://github.com/user-attachments/assets/2c33601b-c34d-45af-9653-c7e9edb9889b">

Reply:
Let's tackle this in a separate ticket 🫡